### PR TITLE
Todo class fetch

### PR DESCRIPTION
**Pull request by richerstars**

### CODE REVIEW

File: homework/src/App.js [hunk 1]
```diff
@@ -1,11 +1,116 @@
...
+        });
+    };
+
+    getDateFromApi = async () => {
+        try {
+            this.startLoader();
+            const response = await axios.get(constants.getDataUrl);
+            this.setState({todos: response.data});
+        } catch (error) {
+            // eslint-disable-next-line no-console
```
Comment (nataliazhuravlova) [inline homework/src/App.js]: return false or without catch

File: homework/src/components/List/List.jsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import ListItem from "./ListItem/ListItem";
+import './List.css';
+
+
```
Comment (nataliazhuravlova) [inline homework/src/components/List/List.jsx]: empty

File: homework/src/App.js [hunk 3]
```diff
@@ -1,11 +1,116 @@
...
+    };
+
+    render() {
+        return (
+            <div className="mainWrapper">
+                <h1> Todo List</h1>
+                <Form addItem={this.addDateToApi}/>
+                {this.state.isLoading ?
+                    <Loader/> :
+                    <List todos={this.state.todos}
```
Comment (nataliazhuravlova) [inline homework/src/App.js]: no data state

File: homework/src/components/List/ListItem/ListItem.jsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const ListItem = ({title, checked, id, deleteTodo, isChecked}) => {
+    const handleDelTodo = () => {
```
Comment (nataliazhuravlova) [inline homework/src/components/List/ListItem/ListItem.jsx]: in one row each handler

File: homework/src/components/Loader/Loader.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import './Loader.css';
+
+const Loader = () => {
```
Comment (nataliazhuravlova) [inline homework/src/components/Loader/Loader.jsx]: without return

File: homework/src/constants/urls.js [hunk 6]
```diff
@@ -0,0 +1,5 @@
+export default {
+    getDataUrl: 'https://jsonplaceholder.typicode.com/todos?_limit=10',
+    deletePutDataUrl: 'https://jsonplaceholder.typicode.com/todos/',
```
Comment (nataliazhuravlova) [inline homework/src/constants/urls.js]: the same url

File: homework/src/App.js (hunk 1 above)
Comment (richerstars) [reply homework/src/App.js]: done

File: homework/src/components/List/List.jsx (hunk 2 above)
Comment (richerstars) [reply homework/src/components/List/List.jsx]: done

File: homework/src/components/List/ListItem/ListItem.jsx (hunk 4 above)
Comment (richerstars) [reply homework/src/components/List/ListItem/ListItem.jsx]: done

File: homework/src/components/Loader/Loader.jsx (hunk 5 above)
Comment (richerstars) [reply homework/src/components/Loader/Loader.jsx]: done

File: homework/src/constants/urls.js (hunk 6 above)
Comment (richerstars) [reply homework/src/constants/urls.js]: done

File: homework/src/components/List/ListItem/ListItem.jsx [hunk 7]
```diff
@@ -1,12 +1,8 @@
 import React from 'react';
 
 const ListItem = ({title, checked, id, deleteTodo, isChecked}) => {
-    const handleDelTodo = () => {
-        deleteTodo(id);
-    };
-    const handleMoveToChecked = () => {
-        checked(id);
-    };
+    const handleDelTodo = () => { deleteTodo(id); };
```
Comment (nataliazhuravlova) [inline homework/src/components/List/ListItem/ListItem.jsx]: without {, const handleDelTodo = () => deleteTodo(id);

File: homework/src/App.js [hunk 8]
```diff
@@ -87,7 +83,7 @@ class App extends Component {
     addDateToApi = async (value) => {
         try {
             this.startLoader();
-            await axios.post(constants.postDataUrl, value);
+            await axios.post(constants.Url, value);
```
Comment (nataliazhuravlova) [inline homework/src/App.js]: Url - url

**Pull request closed without merging**